tox.ini

```diff
@@ -46,11 +47,11 @@ deps =
     doctesting: PyYAML
     numpy: numpy
     pexpect: pexpect
-    pluggymaster: git+https://github.com/pytest-dev/pluggy.git@master
-    pygments
+    pluggymaster: git+https://github.com/pytest-dev/pluggy.git@0.13.1
```

Comment:
Not sure what to do with this - with `master`  pip fails : 
```
The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.1.2 depends on pluggy<1.0 and >=0.12
```

Reply:
The defensiv requirement pin for pluggy may need to be widened

Reply:
I have a couple of questions:
* Is it ok to bump the upper bound for `pluggy`  (there are quite a few commits - [pluggy/compare/0.13.1...master](https://github.com/pytest-dev/pluggy/compare/0.13.1...master))? 
* `pytest>=6.0.0` requirement comes from [pytest-xdist](https://github.com/pytest-dev/pytest-xdist/blob/14bafa6dd5319a83e694cdfea3c2c9d3915c1082/setup.py#L3), which is in tox `deps`. What is the proper way of telling tox/pip to use local pytest (sdist package) for testing, instead of resolving from released ones?

Reply:
The idea of `pluggymaster` is to make sure pluggy master is working. So here it needs to remain `@master`.

But to think the conflict that the new resolver trips over, I think we can bump the `pluggy` requirement in `setup.cfg` to something like `pluggy<1.0.0a1`. pytest works with current pluggy master, and we'll at least try to make sure the next pluggy development release (if any) would use an alpha rather than dev.

Reply:
I have loosened `pluggy` requirements  for pytest locally, but the issue is:
* tox installs `pytest-xdist`
* `pytest-xdist` requires `pytest>=6.0.0`
* pip start resolving `pytest` from **released** versions, instead of local one, that have `pluggy` versions bumped 

I'm just started with `tox` (looking into config options), but perhaps someone have an idea how to solve this already.


Logs from `tox -e py37-pluggymaster-xdist` :
```
ERROR: Cannot install pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master) 
and pytest-xdist because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.1.2 depends on pluggy<1.0 and >=0.12
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.1.1 depends on pluggy<1.0 and >=0.12
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.1.0 depends on pluggy<1.0 and >=0.12
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.0.2 depends on pluggy<1.0 and >=0.12
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.0.1 depends on pluggy<1.0 and >=0.12
    The user requested pluggy 1.0.0.dev42+g0a064fe (from git+https://github.com/pytest-dev/pluggy.git@master)
    pytest 6.0.0 depends on pluggy<1.0 and >=0.12
```

Reply:
adding `xdist: -e .` (7b281fe) seem to work, but not sure if this is the right way.